lib/extras/codec_test.cc

```diff
@@ -396,6 +406,78 @@ TEST(CodecTest, TestWideGamut) {
 
 TEST(CodecTest, TestPNM) { TestCodecPNM(); }
 
+TEST(CodecTest, FormatNegociation) {
```

Comment:
negotiation

Reply:
Thanks, I’m adding this to my list of unnecessary spelling differences between English and French, next to conne**ct**ion/conne**x**ion, distor**t**ion/distor**s**ion, appear**a**nce/appar**e**nce and li**t**era**ll**y/li**tt**éra**l**ement.